### ISSUE TITLE
Documentation tweaks and improvements

### ISSUE THREAD

**Issue by izahn** [hbs-rcs/hbsgrid-docs#43]
The following issues were noted in discussion this morning:

- [ ] Remove top-level blog link
- [ ] Move hbs home link to the top banner
- [ ] Add more direct links (e.g., to dbeaver connection docs, ssl instructions etc.)
- [ ] Info boxes should have informative title, e.g., the one at the top of the research data storage page
- [ ] Add WRDS connection section to database page
- [ ] Look into fixing stata tmp issue

Let me know if I missed anything @velmela

Also I don't think Katie has a github account, or she is not in the hbs-rcs organization. We should help her set that up so she can edit the docs directly.

**Comment by velmela** [comment]
Thanks @izahn! I added @kyosua to hbs-rcs so she should be good to go. Thanks for jotting these down, those match what I had and I'll also add the larger to-do's for Katie and I here so we have everything in one place (and also this task list feature here is very cool):

**Short Term Items**
- [ ] @velmela do a last sweep of the old cluster section of the website to see if there's anything that's not currently on the new site, but should be.
- [ ] @kyosua can you reach out to web services to see if they can link the [Compute Cluster & Data Storage](https://www.hbs.edu/research-computing-services/resources/Pages/default.aspx) tab to the [GitHub page](https://hbs-rcs.github.io/hbsgrid-docs/) starting June 22nd late PM (if possible).
- [ ] Look into the best way to "hide" the old website pages. What happens when someone navigates to a hidden page? (@kyosua let me know how your time looks to check into that--happy to help).

**Medium Term Items**
- [ ] Using SiteImprove, check to see what pages link out to the old Cluster page and adjust as needed.
- [ ] Change the links in the Microsoft Flows to the new pages (@velmela). 
- [ ] Move the [Other Research Computing Environments](https://www.hbs.edu/research-computing-services/resources/other-research-computing-environments.aspx) section on the old site to another place on the old site or somewhere on the new site?
- [ ] Add a link in the Help & Support section in GitHub to the FAQ's on the old site?

**Longer Term/Bigger Picture Items**
- [ ] How much information do we want to include on the new pages around our hostname, etc. for logging into MariaDB?

Please feel free to add anything I missed!

**Comment by izahn** [comment]
I looked at the `STATATMP` issue, not really clear to me what if anything should be done about it. Personally I would just link to https://www.stata.com/support/faqs/data-management/statatmp-environment-variable/

I'm also having second thoughts about the WRDS stuff. The technote is specifically about connecting from Windows, so it doesn't really belong in Grid documentation.

**Comment by velmela** [comment]
Thanks @izahn! @kyosua so you're in the loop--Ista and I spoke on Friday and we landed on a pared down version of the Stata temp page on the new page (which he's created under the troubleshooting section). The WRDS connection section (which based on our monthly metrics is very popular) will remain where it is on the old site under the how-to's.

I looked at the top pages under the resources section from the last 6 months, and a the one thing that caught my eye:

- The "logging in" page is, as you would expect, very popular. I think most of the information there is included in the new page except for some of the instructions for brand new NoMachine users in the local client connection section. May not need it, but thought we could discuss a few specific sections.

Oh snap, doesn't this banner look nice:
![image](https://user-images.githubusercontent.com/42443482/173452863-0ee59e98-ff49-4db8-b13e-6fea4214a44d.png)

**Comment by izahn** [comment]
> Oh snap, doesn't this banner look nice: ![image](https://user-images.githubusercontent.com/42443482/173452863-0ee59e98-ff49-4db8-b13e-6fea4214a44d.png)

Ha, yeah [the first part is an image](https://hbs-rcs.github.io/hbsgrid-docs/imgs/rcs.png), I kind of roughed it. Good enough for now but maybe somebody can make a nicer one.

**Comment by izahn** [comment]
>     * The "logging in" page is, as you would expect, very popular. I think most of the information there is included in the new page except for some of the instructions for brand new NoMachine users in the local client connection section. May not need it, but thought we could discuss a few specific sections.

I think everything is covered in  https://hbs-rcs.github.io/hbsgrid-docs/#quick-start . The series of screenshots in the old documentation has been replaced by the video, and extraneous information has been intentionally omitted. Note that this section will change substantially on June 23rd because [steps 5-7 will no longer be needed](https://github.com/hbs-rcs/hbsgrid-docs/pull/42/files) :-).

**Comment by velmela** [comment]
MV note to self. Pages on the current website that link out to the Compute Section can be found on SiteImprove by navigating to Inventory-->Site Map (or Inventory --> Pages). Can go through the main ones first, then the remainder little by little.

**Comment by velmela** [comment]
- [ ] @izahn in the Quick Start add note next to hostname that everything else (all other defaults) should be kept as is.
- [ ] @izahn add Globus section.
- [ ] @izahn can you check to see if Katie and I can be added to Google Analytics?

**Comment by izahn** [comment]
I added https://hbs-rcs.github.io/hbsgrid-docs/syncfiles/#transfer-data-using-globus , let me know what you think.

**Comment by velmela** [comment]
Thanks @izahn this looks good to me! Two thoughts:

1. What do you guys think about titling that section **Copy, Transfer, & Sync Files** (or something similar) so people know right away that the section includes info on data transfer (since that's what people are usually writing in to ask about, I think they see it less as syncing)?
![image](https://user-images.githubusercontent.com/42443482/173882618-670cdaac-701b-4176-a1b3-55c736f4b64d.png)

2. I don't think we need it right now, but if we get a lot of questions about how to do the Globus transfer after pointing them to the website, we might do a quick video. If we aren't getting a lot of questions about it, though, I'd leave as is.

**Comment by izahn** [comment]
> 1. What do you guys think about titling that section **Copy, Transfer, & Sync Files** 

Haha yeah, for the headlines and menu really we're just trying to find the word that people will understand and go "Ah, *that's what I'm looking for!**". If "transfer" is that word we should just use that. Currently I have "copy and sync" because "sync" has a two-way connotation that I hope gives a hint that you can transfer data both ways. I think "sync" isn't that intuitive or common so I have "copy" in there too.

Bottom line, I like "transfer", but I having all three is too much. "Copy and transfer"? "Transfer and sync"? Definitely overthinking it at this point.

**Comment by velmela** [comment]
Yes, agreed, I think we should go with "that's what I'm looking for!" I vote Copy and Transfer--I think those are the two most commonly used key words.

**Comment by velmela** [comment]
@kyosua just a heads up that this page (www.hbs.edu/research-computing-services/data-practices/Pages/default.aspx) should also get decommissioned during the process of "hiding" pages (it's not technically under the resources tab, so it wouldn't be part of your sweep by default). Thanks!

**Comment by izahn** [comment]
@velmela @kyosua it will be good for both of you to check that you are able to edit the new documentation website directly. Here are a couple typos you could fix as a test to see if if works:

- The "Mount storage locally" infobox at the top of https://hbs-rcs.github.io/hbsgrid-docs/storage/ reads
  > Research storage is also accessible on Windows as a network drive at \\research.hbs.edu and via SMB on OSX/Linux at 
  > smb://research.hbs.edu and via SSH at hbsgrid.hbs.edu.
  
  the first "and" should be removed, and comma's inserted to read
  
  > Research storage is also accessible on Windows as a network drive at \research.hbs.edu, via SMB on OSX/Linux at
  > smb://research.hbs.edu, and via SSH at hbsgrid.hbs.edu.

- The "Home folders" section of  https://hbs-rcs.github.io/hbsgrid-docs/storage/  has a sentence that reads
  > When your space fills up, you will not be able to do any more work, which may lead to programs 
  > acting strangely or crashing altogether, disk error notices, or Input/Output errors
  
  there is no reason for "Input" and "Output" to be capitalized here, this should read 

  > When your space fills up, you will not be able to do any more work, which may lead to programs 
  > acting strangely or crashing altogether, disk error notices, or input/output errors

Do you want to try editing those to see if it works? It will take a few minutes for the changes to appear live on the site.
